psa-crypto/src/types/algorithm.rs

```diff
@@ -42,6 +42,17 @@ pub enum Algorithm {
 
 impl Algorithm {
     /// Check if the algorithm is a HMAC algorithm, truncated or not
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use psa_crypto::types::algorithm::{Algorithm, Mac, FullLengthMac, Hash};
+    /// let hmac = Algorithm::Mac(Mac::Truncated {
+    ///     mac_alg: FullLengthMac::Hmac { hash_alg: Hash::Sha256 },
+    ///     mac_length: 34,
```

Comment:
I'm assuming the `mac_length` is in bytes, not bits, in which case this is invalid because `Sha256` has only 32 bytes (not that I'm saying we shouldn't allow that construction, just saying that maybe something below 32 would be better for an example)

Reply:
Makes sense, will modify!